# Parameters for statistical error simulation
# See: vision_proxy.py
#
# var = data + error
# error = precision*gaus(mu=0, ro=std_dev)

vision:
  precision: 0
  std_dev: 0
